docs/create_resource.md

```diff
@@ -225,4 +225,6 @@ app_genre_book_add:
                 parameters: { title: resource.title }
 ```
 
+Remember that you can use Fully Qualified Class Name (``\App\Controller\BookController``) instead id ``app.controller.book`` 
```

Comment:
I guess we don't need the first slash + "of":

```suggestion
Remember that you can use controller's Fully Qualified Class Name (``App\Controller\BookController``) instead of id ``app.controller.book`` 
```

Reply:
Same in every doc actually.

Reply:
Symfony docs say that as well:

```yaml
services:
    # ...

    # explicitly configure the service
    App\Controller\LuckyController:
```